base/clouddriver/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./clouddriver-caching
+- ./clouddriver-rw
+- ./clouddriver-ro
+- ./clouddriver-ro-deck
+secretGenerator:
+- files: []
+  name: clouddriver-config
+- files: []
```

Comment:
This is embarrassing that I already forget how this works, but is this the place where end users would manually add each user-supplied secret file?

Reply:
No worries, this is a bit confusing!  The way that users would manually add in a user-supplied secret file is in their kustomization that references the base...we create the secret here with no files so that we can reference it in the base `clouddriver` deployment.  But then in the user-specific overlay you can add any files if you use the `merge` operation:
```
secretGenerator:
- files:
  - secrets/k8s-kubeconfig
  name: spinnaker-secrets
  behavior: merge
```